src/utils/schemas/main/CryptoKitties/index.ts

```diff
@@ -0,0 +1,135 @@
+import axios from "axios";
+import { AbiType } from "ethereum-types";
+import {
+  FunctionInputKind,
+  FunctionOutputKind,
+  StateMutability,
+} from "wyvern-js/lib/types";
+import { Schema } from "../../schema";
+import { EventInputKind } from "../../types";
+
+type CryptoKittiesType = string;
+
+export const CryptoKittiesSchema: Schema<CryptoKittiesType> = {
```

Comment:
do we actually need this?

Reply:
We would need this if someone creates an order for CyptoKitties